Sources/Testing/Support/Locked.swift

```diff
@@ -25,7 +25,6 @@ private import TestingInternals
 /// - Bug: The state protected by this type should instead be protected using
 ///     actor isolation, but actor-isolated functions cannot be called from
 ///     synchronous functions. ([83888717](rdar://83888717))
-@propertyWrapper
 struct Locked<T>: @unchecked Sendable where T: Sendable {
```

Comment:
Do we still think the single word "Locked" is a good name for this type, given that it's no longer a property wrapper?

Reply:
I think it's fine, because you'd pronounce it `Locked<Int>` or similar. I'm open to alternatives (`LockedValue<Int>` sounds silly to me though.)